# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1285748123 14400
#      Wed Sep 29 04:15:23 2010 -0400
# Node ID b1ee705aef45420ce9fa1c6c7e236396c7a8a2be
# Parent  b0a9450d81c61694852590b3c4b3a960c985198b
urlwrite.cc: use octave_scalar_map instead of Octave_map

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-09-29  John W. Eaton  <jwe@octave.org>
+
+	* DLD-FUNCTIONS/urlwrite.cc (F__ftp_dir__): Use octave_scalar_map
+	instead of Octave_map.
+
 2010-09-29  John W. Eaton  <jwe@octave.org>
 
 	* DLD-FUNCTIONS/luinc.cc (Fluinc): Use octave_scalar_map
 	instead of Octave_map.
 
 2010-09-29  John W. Eaton  <jwe@octave.org>
 
 	* DLD-FUNCTIONS/__glpk__.cc (F__glpk__): Use octave_scalar_map
diff --git a/src/DLD-FUNCTIONS/urlwrite.cc b/src/DLD-FUNCTIONS/urlwrite.cc
--- a/src/DLD-FUNCTIONS/urlwrite.cc
+++ b/src/DLD-FUNCTIONS/urlwrite.cc
@@ -1107,21 +1107,21 @@ Undocumented internal function\n\
                   if (n == 0)
                     {
                       string_vector flds (5);
                       flds(0) = "name";
                       flds(1) = "date";
                       flds(2) = "bytes";
                       flds(3) = "isdir";
                       flds(4) = "datenum";
-                      retval = Octave_map (flds); 
+                      retval = octave_map (flds); 
                     }
                   else
                     {
-                      Octave_map st;
+                      octave_map st;
                       Cell filectime (dim_vector (n, 1));
                       Cell filesize (dim_vector (n, 1));
                       Cell fileisdir (dim_vector (n, 1));
                       Cell filedatenum (dim_vector (n, 1));
                     
                       st.assign ("name", Cell (sv));
 
                       for (octave_idx_type i = 0; i < n; i++)
